package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "next dev",
     "dev:db:up": "docker-compose -p application-portal -f infrastructure/dev/docker-compose.yaml --env-file .env up -d",
-    "dev:db:down": "docker-compose -p application-portal -f infrastructure/dev/docker-compose.yaml down",
+    "dev:db:down": "docker-compose -p application-portal -f infrastructure/dev/docker-compose.yaml --env-file .env down",
```

Comment:
merge main into your branch and this change should go away. dont forget to pull to update main first